### ISSUE TITLE
STAR scoring results shouldn't include the color gray

### ISSUE THREAD

**Issue by ArendPeter** [Equal-Vote/star-server#577]
![image](https://github.com/Equal-Vote/star-server/assets/9289903/2dfd78c5-2edb-453a-a131-15eea46f6c90)

**Comment by DubemObi** [comment]
Hello, I'll like to work on this, please assign to me

**Comment by ArendPeter** [comment]
Awesome, I assigned it to you!

And to be clear, the runoff graphs should be 3 colors (blue, green, gray), but the score graphs should rotate between blue, green, and lime (the CHART_COLORS variable)

When you make your PR, feel free to add yourself as a contributor [see example PR](https://github.com/Equal-Vote/star-server/pull/575/files). Your github ID should be 109088909

**Comment by ArendPeter** [comment]
This is the line where I added gray to to the score chart. I added an invisible first row as a hack for the alignment, and I added gray as a dummy color. But I forgot that this adds it to the rotation to show up later in the chart.

https://github.com/Equal-Vote/star-server/blob/3743e2d47ea1396c8b6884b43e375e76d23ccde4/packages/frontend/src/components/util.jsx#L85

I think if you change that line to ``colors.unshift(colors.pop())``, it'll fix this issue. That fix will remove the lime color from the end of the list, and put in in the front as the dummy color.
